test/visualTest.ts

```diff
@@ -202,17 +202,16 @@ namespace powerbi.extensibility.visual.test {
                     DefaultWaitForRender);
             });
 
-            it("data labels should be hidden by default", (done: DoneFn) => {
+            it("data labels should not be hidden by default", (done: DoneFn) => {
```

Comment:
Add more unit-tests for:
+* "Group->Show category" and "Group->Font size" were not working
 +* Add colors selection in settings panel
 +* Add an ability to show values of the inner ring
 +* Add an ability to support decimal formatting

Reply:
Added 2 tests